### PR TITLE
feat: use navigation to redirect

### PR DESCRIPTION
**Pull request by Weii-Dev9**
## Overview
- Reference card [#22](https://trello.com/c/vWONfyD0/22-2-项目跳转方式：navigation)

Why was this PR created / what was done?

- use navigation to redirect

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
1. [x] Makes your PR pass tests?
2. [x] PR attached to trello card
3. [x] Trello card moved into 'Review'

### CODE REVIEW

File: app/src/main/java/com/thoughtworks/androidtrain/helloworld/navigation/Nav_graph.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.thoughtworks.androidtrain.helloworld.navigation
```
Comment (yunlongwen) [inline app/src/main/java/com/thoughtworks/androidtrain/helloworld/navigation/Nav_graph.kt]: class name 应该是 “ NavGraph”？

File: app/src/main/java/com/thoughtworks/androidtrain/helloworld/navigation/Screen.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.thoughtworks.androidtrain.helloworld.navigation
+
+sealed class Screen(val route: String) {
```
Comment (yunlongwen) [inline app/src/main/java/com/thoughtworks/androidtrain/helloworld/navigation/Screen.kt:3]: route 看起来没有用？

Comment (Weii-Dev9) [reply app/src/main/java/com/thoughtworks/androidtrain/helloworld/navigation/Screen.kt:3]: Screen密封类中的嵌套对象使用了route属性
object FriendsCircleScreen : Screen("friend_circle_screen")
object PublicTweetScreen : Screen("public_tweet_screen")

File: app/src/main/java/com/thoughtworks/androidtrain/helloworld/navigation/Nav_graph.kt (hunk 1 above)
Comment (Weii-Dev9) [reply app/src/main/java/com/thoughtworks/androidtrain/helloworld/navigation/Nav_graph.kt]: 是将包名navigation改为NavGraph么？

**Review by yunlongwen — APPROVED** [review]

**Pull request merged**